src/OpenAL/OpenTK.OpenAL/ALC/Interfaces/IALC.cs

```diff
@@ -0,0 +1,12 @@
+using AdvancedDLSupport;
+
+namespace OpenTK.OpenAL.Interfaces
+{
+    /// <summary>
+    /// Defines the public interface of the context in the OpenAL 1.1 library.
+    /// </summary>
+    [NativeSymbols(Prefix = "alc")]
+    internal interface IALC : IContext, IContextDevices, IContextErrors, IContextExtensions, IContextState
```

Comment:
The internal interfaces aren't meant to be consumed individually, but I suppose it couldn't hurt to make them public. ADL itself has been fixed to support internal interfaces.

Reply:
@varon Do you think we should expose the interfaces? Doing so would allow users to activate instances of the interfaces (mapping to the same underlying library pointer), enabling passing around of or specializing to subportions of the API.

It does mean some extra public API surface, but it might be warranted.

Reply:
Let's keep the API surface smaller rather than bigger. It's very easy to open it up later if someone has a compelling use case.